src/components/stickyFooter/StickyFooter.tsx

```diff
@@ -3,15 +3,18 @@ import * as React from 'react';
 import * as styles from './StickyFooter.scss';
 
 export interface IStickyFooterProps {
+    isOpened: boolean;
+    id?: string;
     className?: string;
 }
 
 export class StickyFooter extends React.Component<IStickyFooterProps> {
     static ID = 'StickyFooter';
 
     render() {
+        const {id, className, isOpened} = this.props;
         return (
-            <div id={StickyFooter.ID} className={classNames(styles.stickyFooter, this.props.className)}>
+            <div id={id || StickyFooter.ID} className={classNames(styles.stickyFooter, {[styles.stickyFooterOpened]: isOpened}, className)}>

```

Comment:
```suggestion
            <div id={id} className={classNames(styles.stickyFooter, {[styles.stickyFooterOpened]: isOpened}, className)}>
```
I think it'd be more future-proof to use defaultProps instead. Imagine a case where you would need to reuse the `id` prop somewhere else, you would need to repeat this logic. 

Something like the following would solve this issue:
```ts
static defaultProps: Partial<IStickyFooterProps> = {
    id: StickyFooter.ID,
}
```